arch/arm/Kconfig

```diff
@@ -53,6 +53,12 @@ config ARM_ON_ENTER_CPU_IDLE_HOOK
 	  If needed, this hook can be used to prevent the CPU from actually
 	  entering sleep by skipping the WFE/WFI instruction.
 
+config ARM_ON_EXIT_CPU_IDLE_HOOK
+	bool
+	help
+	  Enables a pssibility to inject SoC-specific code just after WFI/WFE
+	  instructions of cpu idle implementation.
```

Comment:
If we are going with the macro injection the description must be way more informative, see https://github.com/zephyrproject-rtos/zephyr/blob/630555d9f6ead7530b3d1e5a5fba96104607550c/arch/riscv/Kconfig#L63-L79 for reference

Reply:
added detailed description